src/SmartComponents/Vulnerability/VulnerabilityCard.js

```diff
@@ -101,68 +92,52 @@ class VulnerabilityCard extends Component {
                         width={ 700 }
                         data={ chartData }
                     />
-                    {criticalVulnerabilitiesFetchStatus === 'pending' && (<Loading />)}
-                </TemplateCardBody>
-                <Divider/>
+                }
+            </TemplateCardBody>
+            <Divider/>
+            {vulnerabilitiesFetchStatus !== 'fulfilled' ? <Loading /> :
                 <TemplateCardBody isFilled={ false } isHorizontalLayout="true">
                     <NumberDescription
-                        data="NA"
+                        data={ vulnerabilities.recent_cves.last90days }
                         dataSize="md"
                         linkDescription="Last 90 days"
```

Comment:
Is it possible to populate this string for `linkDescription` with the last part of the string for `data` so that it's not written manually?

Reply:
The response returns for example:
```
recent_cves = {
 last30days: 8,
 last7days: 0, 
 last90days: 21
}

```
I can grab the keys from the object (ex `last30days`) and pass it as a key for the translation 
`intl.formatMessage(messages.last30days)`  but that's unsafe because if for any reason the backend changes the response, the `react-intl` will throw an error that can't find the ID to format the message

Reply:
Ok, thats a good point. I think its fine to hardcode it then